### PR TITLE
Fix videoconverter and some checkstyle errors

### PR DESCRIPTION
**Pull request by danielaSantaCruz**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 1]
```diff
@@ -88,7 +89,7 @@ private String changeResolution() {
         if (width > 0 && height > 0) {
             scale = width + ":" + height;
             aspectRatio = ":force_original_aspect_ratio=decrease,pad=";
-            resolutionCommand = "-vf \"scale=" + scale + aspectRatio + scale + ":-1:-1:color=white\"";
+            resolutionCommand = " -b:v 5000k -vf \"scale=" + scale + aspectRatio + scale + ":-1:-1:color=gray\"";
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:92]: :V

File: src/main/java/org/fundacion/jala/converter/core/metadata/ExportTypeFile.java [hunk 2]
```diff
@@ -72,8 +78,13 @@ private void setDefaultNameFileWithTypeOfFile() {
         String name = "";
         boolean write = false;
         for (int i = spellName.length - 1; i >= 0; i--) {
-            if (spellName[i] == '.' && !write) write = true;
-            else if (write) name = spellName[i] + name;
+            if (spellName[i] == '.' && !write) {
+                write = true;
+            } else {
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/metadata/ExportTypeFile.java]: } else { is not necessary

Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/core/metadata/ExportTypeFile.java:83]: Ok, it's solved

**Review by Raymundogs — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/core/parameter/AudioParameter.java [hunk 3]
```diff
@@ -16,7 +16,7 @@
 public class AudioParameter extends Parameter {
     private String format;
     private String bitrate;
-    private String frecuency;
+    private String frequency;
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/parameter/AudioParameter.java:19]: K20_HERTZ is String or can be int

**Review by GustavoZHA — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/core/parameter/AudioParameter.java (hunk 3 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/core/parameter/AudioParameter.java:19]: It has to be a String :)

**Review by JessickaMoya — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/core/metadata/ExportTypeFile.java (hunk 2 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/core/metadata/ExportTypeFile.java]: 😠

**Pull request merged**